web/atlas.json

```diff
@@ -1222,7 +1221,6 @@
 {"id": "twpurr", "submitted_by": "danieru_desu", "name": "Mik-Mik packet", "description": "The packaging design of Mik-Mik Sweetened Milk powder manufactured by Jocker's Food Industry, which has been a childhood snack for Filipinos since the 90's.", "website": "", "subreddit": "/r/Philippines", "center": [432.5, 1617.5], "path": [[408.5, 1600.5], [408.5, 1633.5], [455.5, 1633.5], [455.5, 1600.5]]},
 {"id": "twpuro", "submitted_by": "felipro333", "name": "Flag of Spain", "description": "Flag of Spain, created by the Reddit community. It includes a lot of pixel art of various pieces of culture and history of Spain.", "website": "https://en.wikipedia.org/wiki/Spain", "subreddit": "/r/esPlace", "center": [1145.5, 295.5], "path": [[865.5, 281.5], [865.5, 308.5], [1489.5, 313.5], [1493.5, 309.5], [1494.5, 309.5], [1496.5, 307.5], [1496.5, 306.5], [1495.5, 305.5], [1495.5, 304.5], [1493.5, 302.5], [1493.5, 301.5], [1491.5, 299.5], [1490.5, 299.5], [1489.5, 298.5], [1488.5, 298.5], [1485.5, 295.5], [1485.5, 294.5], [1486.5, 293.5], [1486.5, 292.5], [1488.5, 290.5], [1489.5, 290.5], [1490.5, 289.5], [1490.5, 288.5], [1492.5, 286.5], [1493.5, 286.5], [1495.5, 284.5], [1496.5, 284.5], [1497.5, 283.5], [1499.5, 283.5], [1500.5, 282.5], [1501.5, 282.5], [1502.5, 281.5], [1504.5, 281.5], [1505.5, 280.5], [1505.5, 279.5], [1553.5, 279.5], [1553.5, 280.5], [1713.5, 280.5], [1713.5, 286.5], [1718.5, 286.5], [1719.5, 287.5], [1721.5, 287.5], [1723.5, 289.5], [1723.5, 302.5], [1721.5, 304.5], [1715.5, 304.5], [1715.5, 309.5], [1693.5, 309.5], [1693.5, 280.5], [1628.5, 280.5], [1628.5, 308.5], [1595.5, 308.5], [1595.5, 280.5], [1553.5, 280.5], [1553.5, 279.5]]},
 {"id": "twq8ws", "submitted_by": "SirOllie_", "name": "Aqueduct of Segovia", "description": "Here is one of the best-preserved elevated Roman aqueducts in the world, built around the second half of the 1st century A.D. It carried water to the Spanish city of Segovia. The most visible part, and therefore famous, is the arcade that crosses the Plaza del Azoguejo, in the city.", "website": "https://www.wmf.org/project/aqueduct-segovia", "subreddit": "/r/esPlace", "center": [1299.5, 305.5], "path": [[1267.5, 297.5], [1268.5, 313.5], [1329.5, 313.5], [1331.5, 297.5]]},
-{"id": "twpu8y", "submitted_by": "bkc56", "name": "Starlight Glimmer cutie mark", "description": "Represents Starlight Glimmer's equals sign cutie mark from the episode in My Little Pony Friendship is Magic where she was first introduced. The colors are from her mane/tail.", "website": "https://mlp.fandom.com/wiki/Starlight_Glimmer", "subreddit": "/r/MyLittlePony", "center": [900.5, 1747.5], "path": [[898.5, 1746.5], [898.5, 1748.5], [901.5, 1748.5], [901.5, 1746.5]]},

```

Comment:
This is a valid entry, but the outline is too small. Instead of removing it, redraw the outline to show the full equals sign.

Reply:
Alright, I'll update the boundaries instead